Java/CoreNative/src/main/java/com/breadwallet/corenative/support/BRCryptoData16.java

```diff
@@ -1,47 +1,52 @@
 /*
  * Created by Michael Carrara <michael.carrara@breadwallet.com> on 7/1/19.
  * Copyright (c) 2019 Breadwinner AG.  All right reserved.
-*
+ *
  * See the LICENSE file at the project root for license information.
  * See the CONTRIBUTORS file at the project root for a list of contributors.
  */
 package com.breadwallet.corenative.support;
 
+
 import com.sun.jna.Pointer;
 import com.sun.jna.Structure;
 
 import java.util.Arrays;
 import java.util.List;
 
-public class UInt512 extends Structure {
+public class BRCryptoData16 extends Structure {
```

Comment:
You didn't like it in bits?

Reply:
Actually, that was GIT corresponding two files - that aren't really related.  I deleted UInt512.
I added BRCryptoData16 (because UInt128 was in the BRCryptoPeer.h interface).

Reply:
I took the cues from bitcoin, `uint8_t *md32`, ...

Reply:
I'm good with it; just picking your brain on thought process.